### PR TITLE
Dr 24 05 ba vet132

### PR DESCRIPTION
**Pull request by thea-m**
I had some Latin troubles in the description of the collation and one decoration (original commented out in the file)

### CODE REVIEW

File: VaticanBAV/et/BAVet132.xml [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+                            </dimensions>
+                            </extent>
+                          
+                            <collation>
+                              <note>According to the catalogue, the manuscript consists of 19 quires and one 
+                              protective bifolio. The third, twelfth and penultimate quire are quaternions, the 
+                              fourth quire has 9 folios and the last quire has 5 folios. The other quires are
+                              quinions. This description does not seem to add up to 128 folios.</note>
+                              <signatures>No quire marks.</signatures>
+                              <!--constat fasciculis XIX sine notis, bifolio pro custodia prae-
```
Comment (eu-genia) [inline VaticanBAV/et/BAVet132.xml:195]: the error must be in total number
to get 128 we must have 
1x2 (protective)
3x8 (quires 3, 12, 14) 
1x9 (4)
1x5 (15)
10x10 (1, 2, 5-11, 13)
so a total of XVI / 16 quires (or 15 if the protective bifolium is not counted in)

File: VaticanBAV/et/BAVet132.xml [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+                                <item xml:id="q3" n="2">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="13" to="22"/>
+                                </item>
+                                <item xml:id="q4" n="3">
+                                  <dim unit="leaf">8</dim>
+                                  <locus from="23" to="30"/>
+                                </item>
+                                <item xml:id="q5" n="4">
+                                  <dim unit="leaf">9</dim>
```
Comment (eu-genia) [inline VaticanBAV/et/BAVet132.xml:216]: the app won't be able to process it unless a s.l. is mentioned

File: VaticanBAV/et/BAVet132.xml [hunk 3]
```diff
@@ -0,0 +1,424 @@
...
+                                </item>
+                                <item xml:id="q14" n="13">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="117" to="126"/>
+                                </item>
+                                <item xml:id="q15" n="14">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="127" to="136"/>
+                                </item>
+                                <item xml:id="q16" n="15">
```
Comment (eu-genia) [inline VaticanBAV/et/BAVet132.xml:259]: see above,

File: VaticanBAV/et/BAVet132.xml [hunk 4]
```diff
@@ -0,0 +1,424 @@
...
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="137" to="146"/>
+                                </item>
+                                <item xml:id="q17" n="16">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="147" to="156"/>
+                                </item>
+              <item xml:id="q18" n="17">
```
Comment (eu-genia) [inline VaticanBAV/et/BAVet132.xml:267]: 14

File: VaticanBAV/et/BAVet132.xml [hunk 5]
```diff
@@ -0,0 +1,424 @@
...
+                <dim unit="leaf">8</dim>
+                <locus from="157" to="164"/>
+              </item>
+              <item xml:id="q19" n="18">
```
Comment (eu-genia) [inline VaticanBAV/et/BAVet132.xml:271]: 15

File: VaticanBAV/et/BAVet132.xml [hunk 6]
```diff
@@ -0,0 +1,424 @@
...
+                      
+                      <decoDesc>
+                        <decoNote xml:id="d1" type="band">
+                          <desc>Ornamental bands in black and red are frequently present at the beginning of
+                          textual units and after almost every ten psalms.</desc>
+                        </decoNote>
+                        
+                        <decoNote xml:id="d2" type="printedImage" cert="low">
+                          <desc>At the beginning of the Psalter, a strip is covered by a piece of paper showing flowers and pink leaves
+                            on a blue page.</desc>
```
Comment (eu-genia) [inline VaticanBAV/et/BAVet132.xml:310]: turquoise?

Comment (eu-genia) [reply VaticanBAV/et/BAVet132.xml:310]: is by any chance fascia the term they otherwise use for band/harag?

**Comment by MagdaKrzyz** [comment]
I have entered Ms. (BM)[BAVet132] but I received an answer that the item is not there. Also whan I open the BM I cannot see it. What do I do wrong?

**Comment by eu-genia** [comment]
It is not there as long as this PR has not been merged

File: VaticanBAV/et/BAVet132.xml (hunk 2 above)
Comment (thea-m) [reply VaticanBAV/et/BAVet132.xml:216]: I know, but what is the convention for cases in which we don't know which leaf is the single one?

File: VaticanBAV/et/BAVet132.xml (hunk 6 above)
Comment (thea-m) [reply VaticanBAV/et/BAVet132.xml:310]: No, I don't think so :(

File: VaticanBAV/et/BAVet132.xml (hunk 2 above)
Comment (eu-genia) [reply VaticanBAV/et/BAVet132.xml:216]: we can write something (like for the middle, s.l. 5 no stub) and leave a comment that we have not checked

File: VaticanBAV/et/BAVet132.xml (hunk 6 above)
Comment (eu-genia) [reply VaticanBAV/et/BAVet132.xml:310]: was just trying to make sense, cannot visualize what they meant)) very weird phrase, but the translation seems correct. we have to wait till they have digitized ))

File: VaticanBAV/et/BAVet132.xml (hunk 1 above)
Comment (thea-m) [reply VaticanBAV/et/BAVet132.xml:195]: I'm sorry, it still doesn't add up to me. I think I'll leave it commented out

Comment (eu-genia) [reply VaticanBAV/et/BAVet132.xml:195]: sorry too many quinions )) if they do not count the protective sheets then it must be 9x10 (9 quinions) to get 128+2

Comment (CarstenHoffmannMarburg) [reply VaticanBAV/et/BAVet132.xml:195]: Yes, "constat fasciculis XIV" would be correct to meet the total number of 128 folios of the codex.

File: VaticanBAV/et/BAVet132.xml [hunk 7]
```diff
@@ -0,0 +1,412 @@
...
+                                  <locus from="100" to="109"/>
+                                </item>
+                                <item xml:id="q13" n="12">
+                                  <dim unit="leaf">8</dim>
+                                  <locus from="110" to="117"/>
+                                </item>
+                                <item xml:id="q14" n="13">
+                                  <dim unit="leaf">10</dim>
+                                  <locus from="118" to="127"/>
+                                </item>
```
Comment (CarstenHoffmannMarburg) [inline VaticanBAV/et/BAVet132.xml:254]: q14 needs to be deleted to achieve the total number of 14 quires and 128(+2) folios.

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Review by eu-genia — APPROVED** [review]

**Pull request merged**